spatialdata/_core/_transform_elements.py

```diff
@@ -206,7 +207,7 @@ def _(data: DaskDataFrame, transformation: BaseTransformation) -> DaskDataFrame:
         indices = xtransformed["dim"] == ax
         new_ax = xtransformed[:, indices].data.flatten()
         # mypy says that from_array is not a method of DaskDataFrame, but it is
-        transformed[ax] = da.from_array(np.array(new_ax))  # type: ignore[attr-defined]
+        transformed[ax] = dd.from_dask_array(da.from_array(np.array(new_ax)))  # type: ignore[attr-defined]
```

Comment:
I think one is supposed to add a Series, not an array here.

Reply:
I changed this code, now it is fully lazy. I still use a dask array and not a series. I think it works both way, but if there will be performance problems or some other problems I will keep this in mind and maybe check again.